pkgs/applications/science/math/mathematica/10.nix

```diff
@@ -14,6 +15,7 @@
 , xorg
 , libxml2
 , libuuid
+, ...
```

Comment:
```suggestion
```

Reply:
This (and the other ellipses) can't be deleted because there _are_ other arguments passed in. `default.nix` now contains the superset of all arguments from `9.nix`, `10.nix`, `11.nix`, and `generic.nix`, all combined together.